frontend/src/pages/checklist/[filters].tsx

```diff
@@ -245,8 +245,7 @@ const ChecklistResults: FC<ChecklistResultsProps> = ({
               onClick={handlePrint}
               variant="outlined"
               startIcon={<Print />}
-              size="large"
-              className="hidden md:inline-block lg:w-full"
+              className="hidden border border-[rgba(0,0,0,0.12)] p-[6px] md:inline-block"
             >
               {t('print')}
             </Button>
```

Comment:
```suggestion
            <div className="hidden lg:block">
              <Button onClick={handlePrint} variant="outlined" startIcon={<Print />}>
                {t('print')}
              </Button>
            </div>
```

Reply:
The width of the button is different in the mock up.